admin.go

```diff
@@ -174,3 +174,17 @@ func (a *Admin) GenerateLink(ctx context.Context, params GenerateLinkParams) (*G
 
 	return &res, nil
 }
+
+// Delete a user
+func (a *Admin) Remove(ctx context.Context, userID string) (err error) {
```

Comment:
```suggestion
func (a *Admin) Remove(ctx context.Context, userID string) error {
```

I suggest we don't use return values for consistency.